integration-tests/README.MD

```diff
@@ -0,0 +1,40 @@
+## Overview
+
+Integration tests for status-go
+
+## Table of Contents
+
+- [Overview](#overview)
+- [How to Install](#how-to-install)
+- [How to Run](#how-to-run)
+  - [Running Tests](#running-tests)
+- [Implementation details](#implementation-details)
+
+## How to Install
+
+* Install [Docker](https://docs.docker.com/engine/install/) and [Docker Compose](https://docs.docker.com/compose/install/)
+- Checkout all listed contract repositories into one folder: 
+    - [Status Network Token V2](https://github.com/status-im/status-network-token-v2)
+    - [Communities Contracts](https://github.com/status-im/communities-contracts?tab=readme-ov-file#installation-and-development)
+* Add environment variable, path to contract repos e.g. `export PATH_TO_CONTRACT_REPOS=~/directory-with-every-contract-repo`
+* Install [Python 3.10.14](https://www.python.org/downloads/)
+* In `integration-tests/tests` directory, run `pip install -r requirements.txt`
```

Comment:
please commit the requirements file as it is not yet present

Reply:
thanks, it is present in `integration-tests`, a directory above, will change the readme